subworkflows/nf-core/utils_references/main.nf

```diff
@@ -0,0 +1,52 @@
+// JUST TO TEST THE SUBWORKFLOW
+
+workflow UTILS_REFERENCES {
+    take:
+    references
+    param_file
+    param_value
+    attribute_file
+    attribute_value
+    basepath
+
+    main:
+    // GIVING up writing a test for the functions, so writing a subworkflow to test it
+    references_file = extract_references_file(references, param_file, attribute_file, basepath)
+    references_value = extract_references_value(references, param_value, attribute_value)
+
+    emit:
+    references_file
+    references_value
+}
+
```

Comment:
```suggestion
```

This is totally useless, but that was the only way I could find to actually test the functions